examples/waldo/Cargo.toml

```diff
@@ -15,6 +15,7 @@ waldo-core = { path = "core" }
 waldo-methods = { path = "methods" }
 
 [features]
-cuda = ["risc0-zkvm/cuda"]
 default = []
-metal = ["risc0-zkvm/metal"]
+cuda = ["risc0-zkvm/cuda"]
+prove = ["risc0-zkvm/cuda"]
```

Comment:
I don't understand this one

Reply:
In [e975275](https://github.com/risc0/risc0/pull/1065/commits/e97527580f96131e1eecd62cb78eff6468cc5134) I added a passthrough `profiler` feature flag to all the examples so it is easy for folks to try taking a profile of any of them. I saw that the `metal` is still there even though the feature is broken, so I removed it.

Happy to revert this change if it doesn't seem like the right call.